### PR TITLE
[16.0][ADD] account_analytic_distribution_manual: New module to easily auto…

### PR DESCRIPTION
**Pull request by carlos-lopez-tecnativa**
…complete analytic accounts on any model that has a field for analytic accounts.

@Tecnativa TT50322

### CODE REVIEW

File: account_analytic_distribution_manual/security/analytic_security.xml [hunk 1]
```diff
@@ -0,0 +1,9 @@
+<?xml version="1.0" encoding="utf-8" ?>
+<odoo noupdate="1">
+    <record id="analytic_distribution_manual_comp_rule" model="ir.rule">
+        <field name="name">Analytic Manual Distribution multi company rule</field>
+        <field name="model_id" ref="model_account_analytic_distribution_manual" />
+        <field eval="True" name="global" />
```
Comment (pedrobaeza) [inline account_analytic_distribution_manual/security/analytic_security.xml]: Remove this line. It's a compute field.

**Review by pedrobaeza — APPROVED** [review]
Tested on runboat. Fix the question with the tests and that's it!

**Comment by carlos-lopez-tecnativa** [comment]
> Tested on runboat. Fix the question with the tests and that's it!

Done

File: account_analytic_distribution_manual/readme/CONFIGURE.rst [hunk 2]
```diff
@@ -0,0 +1,6 @@
+#. Go to Invoicing > Analytic Accounting > Manual analytic distributions
```
Comment (carolinafernandez-tecnativa) [inline account_analytic_distribution_manual/readme/CONFIGURE.rst]: Go to Invoicing > Configuration > Analytic Accounting > Manual Analytic Distributions

File: account_analytic_distribution_manual/views/account_analytic_distribution_manual_views.xml [hunk 3]
```diff
@@ -0,0 +1,91 @@
...
+                        <label for="name" />
+                        <h1>
+                            <field name="name" placeholder="Name..." />
+                        </h1>
+                    </div>
+                    <group colspan="2">
+                        <field name="active" invisible="1" />
+                        <field name="company_id" groups="base.group_multi_company" />
+                        <field
+                            name="analytic_distribution"
```
Comment (carolinafernandez-tecnativa) [inline account_analytic_distribution_manual/views/account_analytic_distribution_manual_views.xml:42]: I thinks it should be required, as it not make any sense to create a configuration without analytic distribution

File: account_analytic_distribution_manual/views/account_analytic_distribution_manual_views.xml [hunk 4]
```diff
@@ -0,0 +1,91 @@
...
+                />
+            </search>
+        </field>
+    </record>
+
+    <record
+        id="action_account_analytic_distribution_manual"
+        model="ir.actions.act_window"
+    >
+        <field name="name">Manual analytic distributions</field>
```
Comment (carolinafernandez-tecnativa) [inline account_analytic_distribution_manual/views/account_analytic_distribution_manual_views.xml]: Change to Manual Analytic Distributions in all cases

File: account_analytic_distribution_manual/models/account_analytic_distribution_manual.py [hunk 5]
```diff
@@ -0,0 +1,28 @@
...
+    _name = "account.analytic.distribution.manual"
+    _inherit = "analytic.mixin"
+    _description = "Account analytic distribution manual"
+
+    name = fields.Char(required=True)
+    active = fields.Boolean(default=True)
+    company_id = fields.Many2one(
+        "res.company", required=True, default=lambda self: self.env.company
+    )
+
```
Comment (carolinafernandez-tecnativa) [inline account_analytic_distribution_manual/models/account_analytic_distribution_manual.py:14]: When creating Analytic Manual distribution, should we control that percentage cannot be greather than 100?
Please check this example:

image

**Review by carolinafernandez-tecnativa — COMMENTED** [review]
Translation is missing for this new module.

File: account_analytic_distribution_manual/models/account_analytic_distribution_manual.py (hunk 5 above)
Comment (carlos-lopez-tecnativa) [reply account_analytic_distribution_manual/models/account_analytic_distribution_manual.py:14]: Odoo does not have this constraint in analytic models. @pedrobaeza , what do you think?

Comment (pedrobaeza) [reply account_analytic_distribution_manual/models/account_analytic_distribution_manual.py:14]: It's OK as is. It's on analytic distributed nature to allow more than 100%.

**Review by carolinafernandez-tecnativa — APPROVED** [review]
Functional review LGTM Thanks :)

**Pull request merged**